custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -321,18 +340,33 @@ function DownloadController($scope, $rootScope, $location, locationHierarchy, lo
                 }
             });
             vm.selectedMonth = vm.selectedMonth >= 7 ? vm.selectedMonth : 7;
+        } else if((year.id === 2019 && vm.isPPRSelected())) {
+            vm.months = _.filter(vm.monthsCopy, function (month) {
+                    return month.id >= 4
+            });
+            vm.selectedMonth = vm.selectedMonth >= 4 ? vm.selectedMonth : 4;
+            vm.quarters = vm.quartersCopy.slice(1,4);
+            vm.selectedQuarter = vm.selectedQuarter >= 2 ? vm.selectedQuarter : 2;
+
         } else if (year.id === latest.getFullYear()) {
+            const maxQuarter = Math.floor((latest.getMonth() + 1)/4);
```

Comment:
hey, this would be zero is latest.getMonth() is first quarter no?

Reply:
it won't come to that place, because when ever you select the PPR it would omit the latest year if it's the current month is among the starting three months